### PR TITLE
solved bug in add images fragment

### LINKED ISSUE #606: Bug in Add Images Fragment
`Create PDF` button becomes blue but does not get Enabled When Images Added.
`Create PDF` button becomes blue & Enabled after only the File is selected i.e. without adding images.

### Steps to reproduce
* Add Images
* Click on `ADD IMAGES` or `SELECT A FILE`

### Expected behaviour
The process should be in a flow 
* `Select A File` (Already enabled and Blue) others button should Disabled and Gray
* After file selection `Add Images` becomes Enabled and Blue 
* After adding images `Create PDF ` should be Enabled and should become Blue.

### Actual behaviour
`Select A File` and `Add Images` are Enabled and Blue 
`Create PDF` button becomes blue but does not get Enabled When Images Added.
`Create PDF` button becomes blue & Enabled after only the File is selected i.e. without adding images.

### LINKED ISSUE #635: Bug in Filter Images screen
### Steps to reproduce
* Images to PDF
* Select Images
* Open Filter Images
* Apply any filter
* Click Reset
* Click Next/Previous

### Expected behaviour
On  Next/Previous Click
* Snackbar should not be displayed if `reset` is clicked after changes
* Next/Previous image should be displayed.

### Actual behaviour
Snackbar opens with message "Save current image first"

### PR DESCRIPTION
**Pull request by pankajpoply45**
# Description

Fixed the bug in add images fragment.

Fixes #606  
Fixes #635

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by vyankatesh24** [comment]
@pankajpoply45 
Add issue no in the above template below `Fixes #(issue)`

File: app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java [hunk 1]
```diff
@@ -122,6 +122,7 @@ void onViewFilesClick(View view) {
     public void showFileChooser() {
         startActivityForResult(mFileUtils.getFileChooser(),
                 INTENT_REQUEST_PICKFILE_CODE);
+
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java]: Remove this extra change.

Comment (vyankatesh24) [reply app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java]: Working is Fine :+1:

**Comment by pankajpoply45** [comment]
Fixes #641

**Comment by sidhuparas** [comment]
@pankajpoply45 Please fill in the template before. Describe the change this PR is making, which issue does it fix (mention that in template). Tick the tests and checklist. For now they aren't ticked.

**Comment by pankajpoply45** [comment]
@sidhuparas done with filling the template.

**Comment by sidhuparas** [comment]
@pankajpoply45 Revert the latest changes. One PR should only fix/improve/add one feature. You are solving two issues in one PR.

**Comment by Swati4star** [comment]
@sidhuparas Are both the issues getting resolved by this PR?
We can have 2 issues fixed in a single PR :)

**Comment by sidhuparas** [comment]
Yes @Swati4star 2nd issue is fixed as well by this PR. Great job @pankajpoply45 👍

**Comment by pankajpoply45** [comment]
Thank You @sidhuparas . GSSOC is a very good initiative for beginners . Very Happy to be a part of this.

**Pull request merged**